CMakeLists.txt

```diff
@@ -9,6 +9,12 @@ endif()
 find_package(Kokkos 4.3 REQUIRED CONFIG)
 message(STATUS "Found Kokkos: ${Kokkos_DIR} (version \"${Kokkos_VERSION}\")")
 
+# We use minimum compiler versions from Kokkos, with the exception for NVCC (>= 11.5)
+SET(ARBORX_NVCC_MINIMUM 11.5.0)
+if(Kokkos_CXX_COMPILER_ID STREQUAL "NVIDIA" AND Kokkos_CXX_COMPILER_VERSION VERSION_LESS ${ARBORX_NVCC_MINIMUM})
```

Comment:
```suggestion
if(Kokkos_CXX_COMPILER_ID STREQUAL "NVIDIA" AND Kokkos_CXX_COMPILER_VERSION VERSION_LESS 11.5.0)
```

Reply:
I haven't thought too much about it but I wonder if we should be looking at CMAKE_CXX_COMPILER_ID too.
(not blocking though)